lib/dateTime.js

```diff
@@ -738,6 +738,18 @@ export function getDateTimeDescriptor() {
 			weekendStartDay = 4;
 			weekendEndDay = 5;
 			break;
+		case 'cy':
+			dateFormats = ['dddd, d MMMM yyyy', 'dd MMMM yyyy', 'dd/MM/yyyy', 'MMMM yyyy', 'd MMMM', 'd MMM'];
+			months = [
+				['Ionawr', 'Chwefror', 'Mawrth', 'Ebrill', 'Mai', 'Mehefin', 'Gorffennaf', 'Awst', 'Medi', 'Hydref', 'Tachwedd', 'Rhagfyr'],
+				['Ion', 'Chwe', 'Maw', 'Ebr', 'Mai', 'Meh', 'Gor', 'Awst', 'Medi', 'Hyd', 'Tach', 'Rhag']
```

Comment:
It looks like `Chwe` should just be `Chw` (unless I'm looking at the wrong section of the table which is possible since it's giant)

Reply:
I ignored the CLDR for this stuff and pulled these from the monolith here:
https://search.d2l.dev/xref/lms/lp/_lang/LangTerms/translations/cy-gb/Standard.xml?r=760bcabb#1463

Not to say there aren't mistakes in there as I did find a legit one that I sent to be addressed.